---
string[] name
iai_bullet_sim/RigidBody[] body